servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl

```diff
@@ -1874,6 +1874,13 @@ void fragment_shader(in SceneData scene_data) {
 		vec3 ref_vec = normalize(reflect(-view, bent_normal));
 		ref_vec = mix(ref_vec, bent_normal, roughness * roughness);
 
+		uint sorted_reflection_indexes[32];
+		int sorted_count = 0;
+
+		for (uint i = 0; i < 32; i++) {
```

Comment:
Nitpick, you're clearing the array here, but not in the mobile version of the shader. 
I think clearing it is not required as you're never reading from an index that you haven't written to first.

Reply:
Makes sense! I've removed that clearing step. :)